pkg/iommu/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/harvester/pcidevices/iommu
```

Comment:
We don't need a new module here. Just introduce it as a sub-package `"github.com/harvester/pcidevices/pkg/iommu"` (also the module path differs from it's github path, making it hard for external packages to import it).

If we are not confident (yet) exposing it as a package, we can put it under `"github.com/harvester/pcidevices/internal"`.